dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -0,0 +1,969 @@
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileFriendlyName=$UtilMethods.espaceForVelocity($!{")
+                        .append(field.variable())
+                        .append("Object.getFriendlyName()} ))");
+
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FilePath=$UtilMethods.espaceForVelocity($!{")
+                        .append(field.variable())
+                        .append("Object.getPath()}) )");
+                    sb.append("#set($")
+                        .append(field.variable())
+                        .append("FileName=$UtilMethods.espaceForVelocity($!{")
+                        .append(field.variable())
+                        .append("Object.getFileName()} ))");
+                }
+            } // http://jira.dotmarketing.net/browse/DOTCMS-2178
+            else if (field instanceof BinaryField) {
+                java.io.File binFile;
+                String fileName = "";
+                String filesize = "";
+                try {
+                    binFile = content.getBinary(field.variable());
+                    if (binFile != null) {
+                        fileName = binFile.getName();
+                        filesize = FileUtil.getsize(binFile);
+                    }
+                } catch (IOException e) {
+                    Logger.error(this.getClass(), "Unable to retrive binary file for content id " + content.getIdentifier()
+                            + " field " + field.variable(), e);
+                    continue;
+                }
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("BinaryFileTitle=\"")
+                    .append(UtilMethods.espaceForVelocity(fileName))
+                    .append("\" )");
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("BinaryFileSize=\"")
+                    .append(UtilMethods.espaceForVelocity(filesize))
+                    .append("\" )");
+                String binaryFileURI = fileName.length() > 0
+                        ? UtilMethods.espaceForVelocity("/contentAsset/raw-data/" + content.getIdentifier() + "/"
+                                + field.variable() + "/" + content.getInode())
+                        : "";
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("BinaryFileURI=\"")
+                    .append(binaryFileURI)
+                    .append("\" )");
+            } else if (field instanceof SelectField) {
+                sb.append("#set($")
+                    .append(field.variable())
+                    .append("SelectLabelsValues=\"")
+                    .append(field.values()
+                        .replaceAll("\\r\\n", " ")
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "\\r\\n" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)